lib/api/CDataFrameTrainBoostedTreeRunner.cc

```diff
@@ -296,6 +305,12 @@ const std::string CDataFrameTrainBoostedTreeRunner::FEATURE_BAG_FRACTION{"featur
 const std::string CDataFrameTrainBoostedTreeRunner::NUMBER_FOLDS{"number_folds"};
 const std::string CDataFrameTrainBoostedTreeRunner::NUMBER_ROUNDS_PER_HYPERPARAMETER{"number_rounds_per_hyperparameter"};
 const std::string CDataFrameTrainBoostedTreeRunner::BAYESIAN_OPTIMISATION_RESTARTS{"bayesian_optimisation_restarts"};
+const std::string CDataFrameTrainBoostedTreeRunner::TOP_SHAP_VALUES{"top_shap_values"};
+const std::string CDataFrameTrainBoostedTreeRunner::SHAP_PREFIX{"shap_"};
```

Comment:
Maybe
```suggestion
const std::string CDataFrameTrainBoostedTreeRunner::SHAP_PREFIX{"shap."};
```
This way they translated into nested fields and this matches outlier detection feature influence.

Reply:
Fixed, also I moved the constant into `CDataFrameRegressionModel`